packages/protocol/contracts/libs/LibConstants.sol

```diff
@@ -17,7 +17,6 @@ library LibConstants {
     // This number is calculated from K_MAX_NUM_BLOCKS to make
     // the 'the maximum value of the multiplier' close to 20.0
     uint256 public constant K_ZKPROOFS_PER_BLOCK = 1;
-    uint256 public constant K_VERIFICATION_DELAY = 60 minutes;
```

Comment:
why is this removed?

Reply:
I'm not sure, I think daniel updated it in an earlier commit

Reply:
This constant should have been removed in #380

Reply:
I'll mark this resolved then!